fenwick/src/main/java/org/opencadc/fenwick/Main.java

```diff
@@ -170,7 +166,12 @@ public static void main(final String[] args) {
                 System.exit(2);
             }
 
-            InventoryHarvester doit = new InventoryHarvester(daoConfig, resourceID, selector);
+            final String configuredTrackSiteLocations = Main.readProperty(props, TRACK_SITE_LOCATIONS_CONFIG_KEY,
+                                                                          null);
+            final boolean trackSiteLocations = StringUtil.hasText(configuredTrackSiteLocations)
```

Comment:
The way this is handled, trackSiteLocations is optional and defaults to false? I had been trying to maintain a policy of either "configured" or "hard-coded" but no "optional-with-default"... probably I didn't make that clear (it isn't written down). I find "optional-with-default" generally leads to unexpected and therefore wrong behaviour. In this case, one could waste a lot of time and resources building global inventory that didn't know where any files where :-)

Please review the config parsing and make every item required.